docs/ref/applications.txt

```diff
@@ -239,12 +239,10 @@ Methods
     as registering signals. It is called as soon as the registry is fully
     populated.
 
-    You cannot import models in modules that define application configuration
-    classes, but you can use :meth:`get_model` to access a model class by
-    name, like this::
-
-        def ready(self):
-            MyModel = self.get_model('MyModel')
+    This is the first point in the application loading process where it's safe 
+    to import models (see :ref:`how-applications-are-loaded` for more details).
+    You can do so with a regular ``import`` statement or by using
```

Comment:
Yes, "at module level" would be a better wording, good idea.

I'm not sure I would suggest to use import statements, as writing imports inside methods is bad form, but I don't mind very much.

Just keep the idea that you can't import models at module level, because that's what bites people.